core/embed/rust/src/micropython/obj.rs

```diff
@@ -150,10 +168,14 @@ impl TryFrom<Obj> for i64 {
     fn try_from(obj: Obj) -> Result<Self, Self::Error> {
         let mut ll: cty::c_longlong = 0;
 
+        // SAFETY:
+        //  - `ll` is a mutable pointer of the right type.
```

Comment:
`ll` is not a pointer here.

Reply:
fixed in b406bd0a0ff986392c2fccef336a7b9201734023